### ISSUE TITLE
3D window won't close (Debian Squeeze 64 bit 1.0.0RC2 and also in 1.0.0RC3), hangs Slic3r, not recoverable.

### ISSUE THREAD

**Issue by yerazunis** [alexrj/Slic3r#1774]
If I click on "view" (or double-click on the object in the top plate view) I get the 
pannable, zoomable, spinnable 3D window.

But I cannot get rid of that window, and I cannot get back to the main Slic3r window to
actually save the plate, export G-code, etc.

Clicking on the X-windows "X" (to kill the main window) doesn't kill either; I
have to kill from another terminal (or use ^C if launched from bash).

So basically, if I accidentally launch the 3D window, it's a death sentence to whatever
work I've done in Slic3r and not saved by that point.

**Comment by alexrj** [comment]
I'm closing this one as there's no separate 3D window now and we have an issue tracking the problem in the other dialogs (#2688)
